### ISSUE TITLE
how to get actually detected data targets -> batch_outs = f(ins_batch)

### ISSUE THREAD

**Issue by Sandy4321** [fchollet/keras#2000]
output of NN is probability of correct decision
for example 
actual labels:  1  1  0  0  1  0  1  0  1  1
detected:        0  1  0  0  0  1  1  0  1  1
so number of correct decisions is 7
number of mistakes is 3
then probability of correct decision is  3/10 = 0.3, so code gives me this number 0.3, in lstm case is it is named accuracy

but if I know that for first test sample 1 was detects 0 with reliability  0.3 ,
then I may mark this sample as detected with low reliability
Related to this question is question how to get these decisions from NN
so I need these  
detected data:        0  1  0  0  0  1  1  0  1  1

seems to be  it is kind of 
batch_outs = f(ins_batch)
from 
functoin
def _test_loop(self, f, ins, batch_size=128, verbose=0):
from
file models.py

Please make sure that the boxes below are checked before you submit your issue. Thank you!
- [x ] Check that you are up-to-date with the master branch of Keras. You can update with:
  pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps
- [ x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
  pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps
- [x ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by Sandy4321** [comment]
may be to use  model.predict(X_test, verbose=1)

**Comment by Sandy4321** [comment]
maybe to use
model.predict_classes
